Cargo.toml

```diff
@@ -16,4 +16,5 @@ name = "quickcheck"
 [dependencies]
 env_logger = "0.3"
 log = "0.3"
+num = "*"
```

Comment:
Could you please remove the `*` dependency on `num`? Wildcards are not good because it subjects downstream users to arbitrary breakage.